src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -8,94 +8,46 @@
-        role = config["Constants"]["Role"]
-        params_dict = self.setup_cdk_params(config)
-        params_list = []
-        for key, value in params_dict.items():
-            if value:
-                '''
-                TODO: To send json input to typescript code from command line it needs to be enclosed in
-                single-quotes, this is a temp fix to achieve that since the quoted string passed from command line in
-                tesh.sh wrapper script gets un-quoted and we need to handle it here.
-                '''
-                if key == 'additionalConfig':
-                    params_list.append(f" -c {key}=\'{value}\'")
-                else:
-                    params_list.append(f" -c {key}={value}")
-        role_params = (
-            f" --require-approval=never --plugin cdk-assume-role-credential-plugin"
-            f" -c assume-role-credentials:writeIamRoleName={role} -c assume-role-credentials:readIamRoleName={role} "
-        )
-        self.params = "".join(params_list) + role_params
-        self.is_endpoint_public = False
-        self.cluster_endpoint = None
         self.cluster_endpoint_with_port = None
-        self.stack_name = f"opensearch-infra-stack-{self.args.stack_suffix}"
-        if self.manifest:
-            self.stack_name += f"-{self.manifest.build.id}-{self.manifest.build.architecture}"
+        self.password = None
 
     def start(self) -> None:
-        command = f"npm install && cdk deploy \"*\" {self.params} --outputs-file {self.output_file}"
+        command = f"curl -X GET http://{self.args.cluster_endpoint}" if self.args.insecure else f"curl -X GET https://{self.args.cluster_endpoint} -u 'admin:{get_password('2.12.0')}' --insecure"
+        try:
+            result = subprocess.run(command, shell=True, capture_output=True, timeout=5)
+        except subprocess.TimeoutExpired:
+            raise TimeoutError(f"Time out! Couldn't connect to the cluster {self.args.cluster_endpoint}")
 
-        logging.info(f'Executing "{command}" in {os.getcwd()}')
-        subprocess.check_call(command, cwd=os.getcwd(), shell=True)
-        with open(self.output_file, "r") as read_file:
-            load_output = json.load(read_file)
-            self.create_endpoint(load_output)
+        if result.stdout:
+            res_dict = json.loads(result.stdout)
+            self.args.distribution_version = res_dict['version']['number']
         self.wait_for_processing()
-
-    def create_endpoint(self, cdk_output: dict) -> None:
-        loadbalancer_url = cdk_output[self.stack_name].get('loadbalancerurl', None)
-        if loadbalancer_url is None:
-            raise RuntimeError("Unable to fetch the cluster endpoint from cdk output")
-        self.cluster_endpoint = loadbalancer_url
-        self.cluster_endpoint_with_port = "".join([loadbalancer_url, ":", str(self.port)])
+        self.cluster_endpoint_with_port = "".join([self.args.cluster_endpoint, ":", str(self.port)])
```

Comment:
This may cause an issue. We will have to explicitly state that the endpoint should be passed without http or https. 
Verify if benchmark command accepts --target-url in the form of `https://<endpoint>:443`. If yes then ignore else we need to clarify it.